src/rgw/rgw_sal_motr.cc

```diff
@@ -2319,6 +2338,28 @@ int MotrObject::copy_object_same_zone(RGWObjectCtx& obj_ctx,
     *etag = etag_str;
   }
 
+  //Set object tags based on tagging-directive
+  struct req_state* s = static_cast<req_state*>(obj_ctx.get_private());
+  auto tagging_dir = s->info.env->get("HTTP_X_AMZ_TAGGING_DIRECTIVE");
```

Comment:
rename tagging_dir -> tagging_drctv, dir is confusing

Reply:
done